### PR TITLE
Updated licensing requirements

### PR DESCRIPTION
**Pull request by leeford**
Licensing requirements were confusing with mention of:

- The free licensing coming at a later date
- The free licensing model is here
- You need a Phone System as part of an E3 etc.

Have modified a few other points.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/configure-onprem-ra.md [hunk 1]
```diff
@@ -25,10 +25,10 @@ If you have an existing Exchange UM auto attendant and call queue system, before
 If your Phone System service will need a service number, the various dependencies can be met in the following sequence:
 
 1. Obtain a service number
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
1. Obtain a service number.
```

File: Skype/SfbHybrid/hybrid/configure-onprem-ra.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ If you have an existing Exchange UM auto attendant and call queue system, before
...
-2. Buy a Phone System license
+2. Buy a Phone System Virtual User or Phone System license
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
2. Buy a Phone System Virtual User or Phone System license.
```

File: Skype/SfbHybrid/hybrid/configure-onprem-ra.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ If you have an existing Exchange UM auto attendant and call queue system, before
...
 3. Create the resource account. An auto attendant or call queue is required to have an associated resource account.
 4. Wait for an active directory sync between online and on premise
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
4. Wait for an active directory sync between online and on premise.
```

File: Skype/SfbHybrid/hybrid/configure-onprem-ra.md [hunk 4]
```diff
@@ -25,10 +25,10 @@ If you have an existing Exchange UM auto attendant and call queue system, before
...
-5. Assign the Phone System license to the resource account.
+5. Assign the Phone System Virtual User or Phone System license to the resource account.
 6. Assign a service number to the resource account.
 7. Create a Phone System service (a call queue or auto attendant)
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
7. Create a Phone System service (a call queue or auto attendant).
```

Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
7. Create a Phone System service (a call queue or auto attendant)
```

File: Skype/SfbHybrid/hybrid/configure-onprem-ra.md [hunk 5]
```diff
@@ -25,10 +25,10 @@ If you have an existing Exchange UM auto attendant and call queue system, before
...
 8. Associate the resource account with a service: (New-CsApplicationInstanceAssociation)
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-onprem-ra.md]:
```suggestion
8. Associate the resource account with a service: (New-CsApplicationInstanceAssociation).
```

**Comment by scanum** [comment]
@jambirk PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by jambirk** [comment]
closing this, some fixes are missing and others are inappropriate

**Pull request closed without merging**